subsys/net/l2/ethernet/ethernet.c

```diff
@@ -77,12 +77,13 @@ void net_eth_ipv6_mcast_to_mac_addr(const struct in6_addr *ipv6_addr,
 					    sizeof(struct net_eth_addr))); \
 									   \
 		NET_DBG("iface %p src %s dst %s type 0x%x tag %d pri %d "  \
-			"len %zu",					   \
+			"len %zu tag_stripped %s",			   \
 			net_pkt_iface(pkt), log_strdup(out),		   \
 			log_strdup(net_sprint_ll_addr((dst)->addr,	   \
 					    sizeof(struct net_eth_addr))), \
 			type, net_eth_vlan_get_vid(tci),		   \
-			net_eth_vlan_get_pcp(tci), (size_t)len);	   \
+			net_eth_vlan_get_pcp(tci), (size_t)len,		   \
+			tag_stripped ? "yes" : "no");			   \
```

Comment:
Quite minor, but as we know we have limited logging buffer and can lose log messages, I'd suggest to use short strings like "y" or "n" (and then abbreviate heading in format string to e.g. "tagstrip" too).